changelog.d/20240130_093754_carla.martet.ext_implement_autoignored_incidents_output.md

```diff
@@ -0,0 +1,4 @@
+### Changed
+
+- Remove the following attributes from `SCAJSONVulnerablePackageVersion`: `url`, `status`, `ignored_until`, `ignore_reason`, `ignore_comment`.
+- Remove the `source_found` attribute from `SCAJSONScanAllOutput` and `SCAJSONScanDiffOutput`.
```

Comment:
I think changelogs should describe user-visible changes. Here there are none, as these attributes did not exist in the previous ggshield version